code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +20,42 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(user.wear_mask == src && security_lock && locked)
+		if(ishuman(user))
+			if(doUnlock(user))

```

Comment:
Won't this break on unlocked muzzles, since you don't check for `if(!locked)` anywhere?

Also consider gags/tape gags, subtypes of muzzle but don't have security locks.